docs/Configuration.md

```diff
@@ -349,32 +358,31 @@ exporters:
-
-### Failure Time Exporter
+| `NAMESPACES` | no | Restricts the set of namespaces from which metrics will be collected. Ex: `myapp-ns-dev,otherapp-ci` | unset; scans all namespaces |
+| `PELORUS_DEFAULT_KEYWORD` | no | ConfigMap default keyword. Select to use in other data values to indicate `Default Value` should be used | `default` |
 
-The job of the failure time exporter is to capture the timestamp at which a failure occurs in a production environment and when it is resolved.
+### Configuring the Failure Time Exporter
+The Failure Time Exporter captures the timestamp of a failure in a production environment and when it is resolved.
 
-Failure Time Exporter may be deployed with one of three backends, such as JIRA, GithHub Issues and ServiceNow. In one clusters' namespace there may be multiple instances of the Failure Time Exporter for each of the backends or/and watched projects.
+Failure Time Exporter may be deployed with one of three backends: JIRA, GithHub Issues, and ServiceNow. One clusters' namespace can have multiple instances of the Failure Time Exporter for each backend and/or watched projects.
 
-Each of the backend requires specific [configuration](#failureconfigmap), that may be used via ConfigMap associated with the exporter instance.
+Each of the backends requires specific [configurations](#failureconfigmap) using the ConfigMap associated with the exporter instance.
 
-For GitHub Issues and JIRA backends we require that all issues associated with a particular application be labelled with the same `app.kubernetes.io/name=<app_name>` label, or custom label if it was configured via `APP_LABEL`.
+> **Important:** All GitHub Issues and JIRA backend issues associated with a particular application must be labelled with the same `app.kubernetes.io/name=&lt;app_name>` label, or a custom label if it was configured via `APP_LABEL`.
 
 
-#### Instance Config Jira
+#### Instance Config Jira Exporter
+The Instance Config JIRA Exporter expects a specific workflow be used when the issue needs to be `Resolved` with `resolutiondate` and all the relevant issues to be a `Bug` with the `Highest` priority with the `app.kubernetes.io/name=&lt;app_name>` label.
 
-Note: By default JIRA exporter expects specific workflow to be used, where the issue needs to be `Resolved` with `resolutiondate` and all the relevant issues to be type of `Bug` with `Highest` priority with `app.kubernetes.io/name=<app_name>` label. This however can be customized to the orgaization needs by configuring `JIRA_JQL_SEARCH_QUERY`, `JIRA_RESOLVED_STATUS` and `APP_LABEL` options. Please refer to the [Failure Exporter ConfigMap Data Values](#failureconfigmap).
+This exporter can be customized to orgaizational needs by configuring `JIRA_JQL_SEARCH_QUERY`, `JIRA_RESOLVED_STATUS` and `APP_LABEL` options. Refer to the [Failure Exporter ConfigMap Data Values](https://pelorus.readthedocs.io/en/latest/Configuration/#failureconfigmap).
```

Comment:
Internal links should not include full URL, because we loose versions, e.g.:

https://pelorus--603.org.readthedocs.build/en/603/Configuration/#failureconfigmap

Reply:
I deleted "https://pelorus.readthedocs.io/en/latest/Configuration/"...  did I get that right?